raiden/settings.py

```diff
@@ -25,6 +25,7 @@
 }
 
 DEFAULT_RIF_COMMS_GRPC_ENDPOINT = "localhost:5013"
+DEFAULT_RIF_COMMS_GRPC_CLIENT_TIMEOUT_SECONDS = 30
```

Comment:
I dont know which timeout is ok, for example, for me 10 seconds is enough, for connect, subscribe and try to send a message. If the node is taking more that 10 seconds, i think that there is an issue with the transport layer or the rif comms node itself.

So, i think that we can put the less number possible here

I dont know what do you guys think, we can go for

- 5-10 seconds (optimistic approach) 
- 30 seconds (pesimistic approach)
- exponential backoff (we can reuse timeout_exponential_backoff from udp/utils i think)

tbh im ok with 30 seconds for this release, but i want to know your opinions and just give you guys my thoughts

Reply:
I put 30 just because i didn't knew but yeah maybe 10 it's ok or something in the middle like 15

Reply:
lets go for 15